games-arcade/opensyobonaction/opensyobonaction-2.rc2.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="An open-source cross-platform port of the Japanese platformer using SDL"
```

Comment:
Well, I'm affraid haikuporter is more permissive than the guideline, which says the limit is 70 chars:
https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#summary
IMO "open-souce" may be dropped, since that's the case for everything in the tree.
```
SUMMARY="A cross-platform port of the Japanese platformer using SDL"
```
is just below 70 chars. But please feel free to keep the longer summary if you prefer ;-)